app/models/user.rb

```diff
@@ -22,15 +22,14 @@ def self.member(id)
     end
 
     event :accept_rollover_screen do
-      transitions from: %i[transitioned rolled_over], to: :accepted_rollover_2021 do
+      transitions from: %i[notifications_configured transitioned rolled_over], to: :accepted_rollover_2021 do
         guard { FeatureService.enabled?("rollover.can_edit_current_and_next_cycles") }
       end
     end
 
     event :accept_notifications_screen do
-      transitions from: %i[rolled_over accepted_rollover_2021], to: :notifications_configured do
+      transitions from: %i[transitioned rolled_over accepted_rollover_2021], to: :notifications_configured do
```

Comment:
Why is this event still here if we're getting rid of the notifications screen? This event is [fired from the `NotificationsController`](https://github.com/DFE-Digital/publish-teacher-training/pull/1252/files#diff-e7d2878d7ccecd38f6e3a83b5cf85053R23) but I don't see why we still want to do that. This state is strictly about tracking what interrupt pages they're viewed, not tracking whether they've configured notifications, so let's strip the notifications stuff all out.

Reply:
Good point! I'll get the changes working